auto_dev/dao/dummy_data.py

```diff
@@ -1,42 +1,16 @@
 """Generate dummy data for the given models."""
 
 import re
-import random
-from typing import TYPE_CHECKING, Any
+from typing import Any
 
 
-if TYPE_CHECKING:
-    from collections.abc import Callable
-
-
-def generate_ethereum_address() -> str:
-    """Generates a fake Ethereum address."""
-    return "0x" + "".join(random.choices("0123456789abcdef", k=40))  # noqa: S311
-
-
-def generate_boolean() -> bool:
-    """Generates a random boolean value."""
-    return random.choice(["True", "False"])  # noqa: S311
-
-
-def generate_integer() -> int:
-    """Generates a random integer."""
-    return random.randint(1, 100)  # noqa: S311
-
-
-def generate_string() -> str:
-    """Generates a random string."""
-    return "".join(random.choices("abcdefghijklmnopqrstuvwxyz", k=10))  # noqa: S311
-
-
-def generate_number() -> float:
-    """Generates a random float number."""
-    return round(random.uniform(1, 100), 2)  # noqa: S311
-
-
-def generate_chain_id() -> str:
-    """Generates a random chain ID."""
-    return random.choice(["1", "5", "10", "137", "84532", "42161", "10000"])  # noqa: S311
+TYPE_TO_GENERATORS = {
```

Comment:
There is already a mapping from type to example. 
Please use this rather than adding more of the same thing